devel/ansible/roles/bodhi/tasks/main.yml

```diff
@@ -153,7 +153,6 @@
   with_items:
       - bodhi.service
       - fedmsg-tail.service
-      - print-messages.service
```

Comment:
I think it would be nice to keep this service, don't you think?

Reply:
yes actually it does not cost much to leave it